recipes/libstemmer/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set name = "libstemmer" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://snowballstem.org/dist/{{ name }}_c-{{ version }}.tar.gz
+  sha256: b941d9fe9cf36b4e2f8d3873cd4d8b8775bd94867a1df8d8c001bb8b688377c3
+
+build:
+  number: 0
+  script: make
```

Comment:
You need to add a run_exports section which defines what versions downstream packages should depend on after they build against this package.